dev-lang/vala/vala-0.36.8.recipe

```diff
@@ -0,0 +1,133 @@
+SUMMARY="An object-oriented programming language that uses the GObject system"
+DESCRIPTION="Vala is a new programming language that aims to bring modern \
+programming language features to GNOME developers without imposing any \
+additional runtime requirements and without using a different ABI \
+compared to applications and libraries written in C."
+HOMEPAGE="https://wiki.gnome.org/Projects/Vala"
+COPYRIGHT="2006-2016 Jürg Billeter
+	2006-2016 Raffaele Sandrini"
+LICENSE="GNU LGPL v2.1"
+REVISION="1"
+SOURCE_URI="https://download.gnome.org/sources/vala/${portVersion%.*}/vala-$portVersion.tar.xz"
+CHECKSUM_SHA256="df51128ce6ef2505a203c4f17e8269f0edcb5f84619ed562d25a88c52652e5db"
+SOURCE_DIR="vala-$portVersion"
+PATCHES="vala-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="!x86_gcc2 x86"
+
+PROVIDES="
+	vala$secondaryArchSuffix = $portVersion compat >= 0
+	cmd:vala$secondaryArchSuffix
+	cmd:valac$secondaryArchSuffix
+	cmd:vapicheck$secondaryArchSuffix
+	cmd:vapigen$secondaryArchSuffix
+	cmd:vala_gen_introspect$secondaryArchSuffix
+	cmd:vala_${portVersion%.*}$secondaryArchSuffix
+	cmd:vala_gen_introspect_${portVersion%.*}$secondaryArchSuffix
+	cmd:valac_${portVersion%.*}$secondaryArchSuffix
+	cmd:vapicheck_${portVersion%.*}$secondaryArchSuffix
+	cmd:vapigen_${portVersion%.*}$secondaryArchSuffix
+	lib:libvala_${portVersion%.*}$secondaryArchSuffix = 0.0.0 compat >= 0
```

Comment:
@fbrosson didn't test the $libVersion here, but it probably would mean using something like $cmdVersion too (would seen strange to use $libVersion in the cmd:* section) ?

Reply:
When `cmd:` items have a version, it can only be $portVersion. So a $cmdVersion is not necessary on any package.

Reply:
Ok, $libVersion wouldn't do any good here also, as it doesn't provide the .so version, so best leave it as is then? :)

Reply:
I would not be so sure about that. It could make sense to version a command if there are API (well command line interface) changes to it and other packages rely on a specific version. But admitedly this is probably very unusual.